machine_learning_hep/analysis/analyzer_jets.py

```diff
@@ -621,6 +627,8 @@ def _subtract_sideband(self, hist, var, mcordata, ipt):
             self._save_hist(fh_sideband,
                             f'sideband/h_ptjet{label}_sideband_{string_range_pthf(range_pthf)}_{mcordata}.png')
             fh_subtracted.Add(fh_sideband, -1.)
+        else:
+            self.logger.warning("Skipping sideband subtraction for %s %s ipt %d", var, mcordata, ipt)
```

Comment:
For MC we do not have background, wouldn't we now always get a spurious warning for that? I.e. should we not restrict this to data?

Reply:
But the subtraction is implemented for MC and I don't see any explicit switch that would disable it.